plenum/common/ledger_manager.py

```diff
@@ -86,6 +88,20 @@ def addLedger(self, iD: int, ledger: Ledger,
             verifier=MerkleVerifier(ledger.hasher)
         )
 
+    def request_ledger_status_if_needed(self, ledger_id):
+        ledgerInfo = self.getLedgerInfoByType(ledger_id)
+        nodes = [node for node in self.owner.nodeReg if node not in ledgerInfo.ledgerStatusOk]
+        self.owner.request_ledger_status_from_nodes(ledger_id, nodes)
```

Comment:
What will be if we do not gather sufficient count of LedgerStatuses this time too?

Reply:
I suggest that we do this change in a separate task